rotkehlchen/tests/unit/test_accounting.py

```diff
@@ -969,6 +969,68 @@ def test_fees_in_received_asset(accountant):
     assert FVal(report['total_profit_loss']) == FVal(14.13870)
 
 
+@pytest.mark.parametrize('mocked_price_queries', [prices])
+@pytest.mark.parametrize('dont_mock_price_for', [[A_KFEE]])
+def test_kfee_price_in_accounting(accountant):
+    """
+    Test that KFEEs are corrently handled during the accounting
+    """
+    history = [
+        {
+            'timestamp': 1609537953,
+            'base_asset': 'ETH',
+            'quote_asset': A_USDT.identifier,
+            'trade_type': 'sell',
+            'rate': 1000,
+            'fee': '30',
+            'fee_currency': A_KFEE.identifier,
+            'amount': 0.02,
+            'location': 'binance',
```

Comment:
nitpick: Let's make the test believable :wink: 

```suggestion
            'location': 'kraken',
```